packages/core/src/observable/index.ts

```diff
@@ -1,16 +1,10 @@
-export { isObservable, observable } from './observable';
 export type { inferObservableValue } from './observable';
+export { isObservable, observable, observableToPromise } from './observable';
+export { map, share, tap } from './operators';
 export type {
-  MonoTypeOperatorFunction,
   Observable,
   Observer,
-  OperatorFunction,
-  Subscribable,
-  SubscriptionLike,
   TeardownLogic,
-  UnaryFunction,
   Unsubscribable,
   UnsubscribeFn,
 } from './types';
-export { map, share, tap } from './operators';
```

Comment:
How come you prefer to slap all operators in one file? Separated feels cleaner to me

No strong preference but I sorta just copied https://github.com/ReactiveX/rxjs/tree/master/packages/rxjs/src/internal/operators

Reply:
I didnt feel like the observable/internals felt necessary, and since we only had 3 it made sense to just squash it into a single file but can def revert if you prefer

Reply:
i love the internals-folder being removed, i am neutral towards combining the operators